src/components/custom/other-platforms.js

```diff
@@ -207,37 +207,45 @@ export const SmarttraderCard = ({ is_selected, word_break_cover }) => (
     </StyledLink>
 )
 
-export const OtherPlatform = ({ header, subHeader, exclude, is_nav, is_ppc_redirect }) => (
-    <SectionContainer padding="0">
-        {is_nav ? null : (
-            <HeaderWrapper>
-                <Header as="h3" type="section-title" align="center">
-                    {header ? header : localize('Check out our other platforms')}
-                </Header>
-                {subHeader && (
-                    <Header
-                        as="h4"
-                        type="sub-section-title"
-                        align="center"
-                        weight="500"
-                        max_width="67.6rem"
-                        m="0.8rem auto"
-                    >
-                        {localize(
-                            'Whether you’re a beginner or a seasoned trader, our trading experience is something you’ll love.',
-                        )}
+export const OtherPlatform = ({ header, subHeader, exclude, is_nav, is_ppc_redirect }) => {
+    const { is_eu_country } = React.useContext(DerivStore)
+
+    return (
+        <SectionContainer padding="0">
+            {is_nav ? null : (
+                <HeaderWrapper>
+                    <Header as="h3" type="section-title" align="center">
+                        {header
+                            ? header
+                            : is_eu_country
+                            ? localize('Check out our other platform')
+                            : localize('Check out our other platforms')}
```

Comment:
Please try to avoid these nested ternary calls. We can declare outside a function `getHeaderText` that returns the respective text to avoid these.